### ISSUE TITLE
Do I need other plugins?

### ISSUE THREAD

**Issue by eusthace811** [autobotsrocks/cordova-imageBase64#1]
Do I need to add more plugins to my project?
I am getting this error:
FileTransferError {code: 2, source: ""https://firebasestorage.googleapis.com/ba8-4fd5-bfee-393c2088a665"", target: "file:///var/mobile/Containers/Data/Application/F60…3D1FFB/Library/Caches//imageBase64Cache.jpg", http_status: null, body: null…}

Any help?

Thank you!

**Comment by nanjingboy** [comment]
No, you don't need to install other plugins. Could you give me the source image url?
